<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModDownloader</class>
 <widget class="QWidget" name="ModDownloader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>模组下载器</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="gb_search">
     <property name="title">
      <string>搜索</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="l_name">
        <property name="text">
         <string>名称</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="le_name"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="pb_search">
        <property name="text">
         <string>搜索</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QListWidget" name="lw_result"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
